src/main/cpp/render/objects/gl_mesh.cpp

```diff
@@ -15,8 +15,8 @@ namespace nova {
 
     gl_mesh::gl_mesh(mesh_definition &definition) {
         create();
-        set_data(definition.vertex_data, definition.vertex_format, usage::static_draw);
-        set_index_array(definition.indices, usage::static_draw);
+        set_data(definition.vertex_data, definition.vertex_format, usage::dynamic_draw);
```

Comment:
i wanted to find a way to reuse meshes and their buffers but didn't get anywhere. Reverted.